addons/website_slides/static/tests/tours/slide_course_publisher_standard.js

```diff
@@ -100,6 +100,48 @@ wTourUtils.registerWebsitePreviewTour('course_publisher_standard', {
 }],
     slidesTourTools.addImageToSection('Introduction', 'Overview', true),
     slidesTourTools.addPdfToSection('Introduction', 'Exercise', true),
+    slidesTourTools.addQuizToSection('Introduction', 'quiz', true),
+[{
+    content: 'eLearning: add a question',
+    trigger: ':iframe input[name="question-name"]',
+    run: "edit Gender",
```

Comment:
Please ask a question, and choose a topic suitable for a right/wrong setting...
```suggestion
    run: "edit In which year was this course created?",
```